logging/unit_tests/test__gax.py

```diff
@@ -612,56 +612,72 @@ def test_ctor(self):
         self.assertIs(api._client, client)
 
     def test_list_sinks_no_paging(self):
+        import six
         from google.gax import INITIAL_PAGE
-        from google.cloud._testing import _GAXPageIterator
         from google.logging.v2.logging_config_pb2 import LogSink
+        from google.cloud._testing import _GAXPageIterator
+        from google.cloud.logging.sink import Sink
 
         TOKEN = 'TOKEN'
-        SINKS = [{
-            'name': self.SINK_PATH,
-            'filter': self.FILTER,
-            'destination': self.DESTINATION_URI,
-        }]
         sink_pb = LogSink(name=self.SINK_PATH,
                           destination=self.DESTINATION_URI,
                           filter=self.FILTER)
         response = _GAXPageIterator([sink_pb], page_token=TOKEN)
         gax_api = _GAXSinksAPI(_list_sinks_response=response)
-        api = self._makeOne(gax_api, None)
+        client = object()
+        api = self._makeOne(gax_api, client)
 
-        sinks, token = api.list_sinks(self.PROJECT)
+        iterator = api.list_sinks(self.PROJECT)
+        page = six.next(iterator.pages)
+        sinks = list(page)
+        token = iterator.next_page_token
 
-        self.assertEqual(sinks, SINKS)
+        # First check the token.
```

Comment:
Indeed. I have been trying to minimally change the old behavior without thinking about it.